axelor-admin/src/main/java/com/axelor/apps/base/service/app/AppServiceImpl.java

```diff
@@ -564,4 +564,34 @@ public void importRoles() throws AxelorException {
       importRoles(app);
     }
   }
+
+  @Override
+  public String getDataExportDir() throws AxelorException {
+    String appSettingsPath = AppSettings.get().get("data.export.dir");
+    if (appSettingsPath == null) {
+      throw new AxelorException(
+          TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
+          I18n.get(IExceptionMessages.DATA_EXPORT_DIR_ERROR));
+    } else {
+      if (!appSettingsPath.endsWith(File.separator)) {
+        appSettingsPath = appSettingsPath + File.separator;
+      }
+      return appSettingsPath;
```

Comment:
Optimize these lines of code, you can do the same by just one line of code
Same in getFileUploadDir() method